user-group/survey01-email.html

```diff
@@ -1,114 +1,114 @@
 <body style="margin:0;padding:0">
-	<div style="Margin:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;min-width:100%;background-color:#EAECEC;">
-		<table style="border-spacing:0;width:100%;table-layout:fixed;">
-			<tbody>
-				<tr>
-					<td align="center" style="padding-top:30px;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;">
+ <div style="Margin:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;min-width:100%;background-color:#EAECEC;">
+  <table style="border-spacing:0;width:100%;table-layout:fixed;">
```

Comment:
Looks like some sort of error crept in:

<img width="999" alt="screen shot 2017-02-07 at 11 48 12" src="https://cloud.githubusercontent.com/assets/109850/22688172/5fd71b20-ed2b-11e6-8ec0-0b8fa852f5ca.png">